app/controllers/users/approvals_controller.rb

```diff
@@ -1,10 +1,14 @@
 class Users::ApprovalsController < ApplicationController
 
-  before_action :authenticate_user!
+  before_action :authenticate_user! 
+  before_action :check_user, only: :index
 
   def index
     @approved_devs = current_user.approved_developers
     @pending_approvals = current_user.pending_approvals
+    if @pending_approvals.length == 0 && params[:redirect]
+      redirect_to params[:redirect]
```

Comment:
This and the other redirect in check_user are both raising issues in Codeclimate as unprotected redirects. Is this a big issue in this context? If it is what's the best way to solve?